charts/astronomer/templates/houston/houston-upgrade-deployments-job.yaml

```
@@ -33,6 +33,12 @@ spec:
       tolerations:
 {{ toYaml (default .Values.global.platformNodePool.tolerations .Values.tolerations) | indent 8 }}
       restartPolicy: Never
+      serviceAccountName: {{ template "houston.updaterServiceAccount" . }}
+      initContainers:
+        - name: prisma-deployment-rollout-status
+          image: {{ template "houston.image" . }}
+          imagePullPolicy: {{ .Values.images.houston.pullPolicy }}
+          command: ["kubectl", "rollout", "status", "-n astronomer", "--watch", "deployment/astronomer-prisma"]
```

```suggestion
          command: ["kubectl", "rollout", "status", "-n", "astronomer", "--watch", "deployment/astronomer-prisma"]
```

---

```suggestion
          command: ["kubectl", "rollout", "status", "-n {{ .Release.Namespace }}", "--watch", "deployment/{{ .Release.Namespace }}-prisma"]
```

@andriisoldatenko I haven't tried this but we should make these templated. This might work or maybe we need to `printf`.